acceptance/config/aio/options.rb

```diff
@@ -9,8 +9,9 @@
     'setup/common/pre-suite/040_ValidateSignCert.rb',
     'setup/aio/pre-suite/045_EnsureMasterStarted.rb',
   ],
-  :is_puppetserver             => true,
-  :'use-service'               => true, # use service scripts to start/stop stuff
-  :puppetservice               => 'puppetserver',
-  :'puppetserver-confdir'      => '/etc/puppetlabs/puppetserver/conf.d',
+  'is_puppetserver'            => true,
+  'use-service'                => true, # use service scripts to start/stop stuff
+  'puppetservice'              => 'puppetserver',
+  'puppetserver-confdir'       => '/etc/puppetlabs/puppetserver/conf.d',
+  'puppetserver-config'        => '/etc/puppetlabs/puppetserver/conf.d/puppetserver.conf'
```

Comment:
Note we currently only test puppetserver in `ci:test:aio` which is why this options block is here and not in `acceptance/config/common/options.rb`